src/utils.tsx

```diff
@@ -1,17 +1,16 @@
 import React, { Children, ReactNode } from 'react';
-import { StyleSheet, View } from 'react-native';
+import { View } from 'react-native';
 
 export const childrenWithOverriddenStyle = (children?: ReactNode) => {
   return Children.map(children, (child) => {
     const element = child as React.ReactElement;
     return (
       // Add a wrapper to ensure layout is calculated correctly
-      <View style={StyleSheet.absoluteFill} collapsable={false}>
-        {/* @ts-ignore */}
+      <View style={{ height: '100%', width: '100%' }} collapsable={false}>
```

Comment:
Probably we should refine this function, since styles should not be overridden

Reply:
Removed style overwriting. Now each child is only wrapped with wrapper view.